src/index.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+// import "babel-polyfill";
```

Comment:
Did you comment this out on purpose? It's broken in IE because it's missing polyfills.